src/stores/mapStore.ts

```diff
@@ -90,26 +81,15 @@ export class MapStore {
 
     // TODO: move logic out of store? You can setCoordinates() instead
     @action
-    setCoordinatesFromDisplayCoordinateSystem(lat: number, lon: number) {
+    public setCoordinatesFromDisplayCoordinateSystem(lat: number, lon: number) {
         const [wgsLat, wgsLon] = GeometryService.reprojectToCrs(
             lat, lon, CoordinateSystem.EPSG4326, this._displayCoordinateSystem);
         this._coordinates = new LatLng(wgsLat, wgsLon);
     }
 
-    set displayCoordinateSystem(value: CoordinateSystem) {
-        this._displayCoordinateSystem = value;
-    }
-
-    // TODO: move logic out of store? You can use set displayCoordinates() instead
-    @action
-    public cycleCoordinateSystem() {
-        this._displayCoordinateSystem =
-            GeometryService.nextCoordinateSystem(this._displayCoordinateSystem);
-    }
-
     @action
-    setNodeSize(nodeSize: NodeSize) {
-        this._nodeSize = nodeSize;
+    public setDisplayCoordinateSystem(value: CoordinateSystem) {
```

Comment:
If we're really choosing to ban setters, lets have it said in `contributions.md`